client/jetpack-onboarding/steps/site-type.jsx

```diff
@@ -9,15 +9,42 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
+import Main from 'components/main';
+import Tile from 'components/tile-grid/tile';
+import TileGrid from 'components/tile-grid';
 
 class JetpackOnboardingSiteTypeStep extends React.PureComponent {
 	render() {
 		const { translate } = this.props;
 		const headerText = translate( "Let's shape your new site." );
 		const subHeaderText = translate( 'What kind of site do you need? Choose an option below:' );
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		return (
+			<Main>
+				<DocumentHead title={ translate( 'Site Type ‹ Jetpack Onboarding' ) } />
+
+				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+
+				<TileGrid>
+					<Tile
+						buttonLabel={ translate( 'Personal site' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Personal Site' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).